replacy/__init__.py

```diff
@@ -72,11 +104,16 @@ def validate_match_dict(match_dict):
         match_dict_schema = get_match_dict_schema()
         validate(instance=match_dict, schema=match_dict_schema)
 
-    def get_predicates(self, match_hooks):
+    def get_predicates(self, match_hooks) -> List[Callable]:
         predicates = []
         for hook in match_hooks:
-            # template - ex. succeeded_by_word
-            template = getattr(custom_patterns, hook["name"])
+            # template - ex. succeeded_by_phrase
+            try:
+                template = getattr(self.replacy_patterns, hook["name"])
+            except AttributeError:
+                # if the hook isn't in custom_match_hooks, this will still
+                # raise an exception. I think that is the correct behavior
+                template = getattr(self.custom_match_hooks, hook["name"])
```

Comment:
I am finding it quite confusing that `self.replacy_patterns` and `self.custom_match_hooks` are actually of the same type. Just the naming convention ...

Reply:
I totally agree. Maybe `self.default_match_hooks` and `self.custom_match_hooks`?